src/snowflake/snowpark/_internal/analyzer/analyzer_utils.py

```diff
@@ -1025,8 +1025,8 @@ def copy_into_table(
     file_path: str,
     file_format_type: str,
     format_type_options: Dict[str, Any],
-    copy_options: Dict[str, Any],
-    pattern: str,
+    copy_options: Optional[Dict[str, Any]],
```

Comment:
`copy_options` is technically never `None` given https://github.com/snowflakedb/snowpark-python/blob/697294d8ee89396deab4309dea384b1eadb75d6b/src/snowflake/snowpark/dataframe.py#L2800